source/includes/code-examples/FindMany.cs

```diff
@@ -0,0 +1,97 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class FindMany
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        // Find multiple documents using builders
+        WriteLine("Finding documents with builders...:");
+        FindMultipleRestaurantsBuilderSync();
+
+        // Extra space for console readability 
+        WriteLine();
+
+        // Find multiple documents using LINQ
+        WriteLine("Finding documents with LINQ...:");
+        FindMultipleRestaurantsLINQSync();
+
+        WriteLine();
+
+        // Find All restaurants
+        WriteLine("Finding all documents...:");
+        FindAllRestaurantsSync();
+    }
+
+    private static void FindMultipleRestaurantsBuilderSync()
+    {
+        // start-find-builders-sync
+        var filter = Builders<Restaurant>.Filter
+            .Eq(r => r.Cuisine, "Pizza");
+
+        var restaurants = _restaurantsCollection.Find(filter).ToList();
+        // end-find-builders-sync
+
+        WriteLine("Number of documents found: " + restaurants.Count);
+    }
+
+    private static void FindMultipleRestaurantsLINQSync()
+    {
+        // start-find-linq-sync
+        var query = _restaurantsCollection.AsQueryable()
+            .Where(r => r.Cuisine == "Pizza").ToList();
+        // end-find-linq-sync
+
+        WriteLine("Number of documents found: " + query.Count);
+    }
+
+    private static void FindAllRestaurantsSync()
+    {
+        // start-find-all-sync
+        var restaurants = _restaurantsCollection.Find(new BsonDocument()).ToList();
```

Comment:
for empty filter, I would recommend using: `EmptyFilterDefinition<BsonDocument>.Empty`. Same in other places

Reply:
I would suggest `Builders<BsonDocument>.Filter.Empty` is more idiomatic and discoverable.